core/src/main/kotlin/model/Documentable.kt

```diff
@@ -75,7 +76,7 @@ interface WithAbstraction {
     val modifier: Modifier
 
     enum class Modifier {
-        Abstract, Open, Final
+        Abstract, Open, Final, Sealed, Empty
```

Comment:
I went with nullabling Modifiler and it was merged, so you can remove empty and go the same way